imports/ui/DragAndDropArea.jsx

```diff
@@ -0,0 +1,36 @@
+// QLICKER
+// Author: Viraj Bangari
+//
+// A general purpose drag and drop component.
+
+import React, { Component, PropTypes } from 'react'
+import ReactDOM from 'react-dom'
+import Dropzone from 'dropzone'
+
+export class DragAndDropArea extends Component {
+
+  render () {
+    return (
+        this.props.children
+    )
+  }
+
+  componentDidMount () {
+    // Dropzone is a JS library that implements dragging and
+    // dropping. A callback is called to a bound DOM node. By
+    // using ReactDOM.findDOMNode(this), a reference to the
+    // rendered DOM node is bound.
+    this.dropzone = new Dropzone(ReactDOM.findDOMNode(this), {
+      url: this.props.url || '/dummy/url/to/send/post',
+      acceptedFiles: this.props.acceptedFiles || 'image/jpeg,image/png,image/gif',
+      accept: this.props.onDrop
+    })
+  }
+
+}
+
+DragAndDropArea.propTypes = {
```

Comment:
In the future, we'll want this component to be able to upload multiple files (e.g. drag multiple files into the drop area). Should the option of multiple files be passed in as a prop, or can that be handled by the onDrop function that is passed as prop?